xayagame/Makefile.am

```diff
@@ -42,34 +56,39 @@ xayagame_HEADERS = \
   rpc-stubs/xayarpcclient.h \
   rpc-stubs/xayarpcserverstub.h
 
-check_PROGRAMS = tests
-TESTS = tests
+if WINDOWS
```

Comment:
Why do you disable the tests on Windows?  Are they hard to cross-compile?

Reply:
My target goal was to cross-compile core library only, not sure why anyone would need tests to run on windows. But yes, if you think its best to get rid of as much platform-conditional things as possible, I will get rid of those and cross-compile with test.

Reply:
If possible, please compile also the tests.  Ideally, they should even *run* without issues - that's why we have them in the first place! :D

Reply:
ok! =)